docs/source/api/components/step.rst

```diff
@@ -8,6 +8,9 @@ Base class
    :members: 
    :special-members:
```

Comment:
Might want to exclude some inherited methods from being documented.
```suggestion
   :special-members:
   :exclude-members: from_params
```

Reply:
This is a good idea, but `from_params()` already doesn't show up?

Reply:
I guess I would like it to generate a concise list of inherited members, with links, rather than duplicating the docs every time.

Reply:
I see it here in my local build: http://127.0.0.1:8000/api/components/step.html#tango.step.Step.from_params

Reply:
That link obviously goes somewhere else on my box. I happen to have the Sphinx server running though, and it goes to a copy that doesn't have `from_params()` 🤪.

I think I'll take the suggestion anyways and we don't think too hard about why it's different on my box until it happens again?

Reply:
Could be different Sphinx versions? I just updated so I'm using the latest. If you've been running the server for a while you may want to shut it down and clean the docs build directory. `make docs` will automatically clean the build directory before starting the server.